compose/compose-file/compose-file-v2.md

```diff
@@ -1269,7 +1269,40 @@ refer to it within the Compose file:
 
 ## Variable substitution
 
-{% include content/compose-var-sub.md %}
```

Comment:
Not sure about removing included content and adding non-included content here. Can you explain it?